files/create-ecr-repo/src/handler.py

```diff
@@ -100,15 +103,28 @@ def run(event, context):
             sys.exit(1)
 
         try:
-            policy = lifecycle_policy()
-            if policy:
+            lifecycle_policy = get_valid_json_from_env("REPO_LIFECYCLE_POLICY")
+            if lifecycle_policy:
                 client.put_lifecycle_policy(
                     registryId=account_id,
                     repositoryName=repository,
-                    lifecyclePolicyText=policy,
+                    lifecyclePolicyText=lifecycle_policy,
```

Comment:
I guess we could use the Python magic here 😆 , but this is fine

```python
if (lifecycle_policy := get_valid_json_from_env("REPO_LIFECYCLE_POLICY")):
  client.put_lifecycle_policy(
    registryId=account_id,
    repositoryName=repository,
    lifecyclePolicyText=policy,
    lifecyclePolicyText=lifecycle_policy,
  )
```

Reply:
no magic 🧙 just beauty ^_^

Reply:
Oh, my, walrus operator.. It is too close to Golang, but at the same time too far

Reply:
well then, by all means